release/images/Dockerfile-gcloud

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:332.0.0-alpine
```

Comment:
How up-to-date do we want this image? It looks like that image is release weekly and is 6 versions ahead of what we're using here.

https://github.com/GoogleCloudPlatform/cloud-sdk-docker/releases/tag/338.0.0

Reply:
Yeah, Ideally we should update this on every release. The alternative would be to just use the latest cloud-sdk image, but that isn't necessarily a great solution either.